config/config-mp/src/main/java/io/helidon/config/mp/MpEnvironmentVariablesMetaConfigProvider.java

```diff
@@ -42,4 +42,5 @@ public List<? extends ConfigSource> create(String type, Config metaConfig, Strin
     public int priority() {
         return 300;
     }
+
```

Comment:
This change should not be in the file. You will probably need to keep the copyright update, unless you reset your changes.

Reply:
I think the copyright job will fail, unless I squash everything in a single commit. But if I do this, you will have to review everything again.